include/SANDGeoManager.h

```diff
@@ -12,28 +15,45 @@
 
 namespace sand_geometry
 {
+const char* const path_internal_volume =
+    "volWorld_PV/rockBox_lv_PV_0/volDetEnclosure_PV_0/volSAND_PV_0/"
+    "MagIntVol_volume_PV_0/sand_inner_volume_PV_0";
+const char* const name_internal_volume = "sand_inner_volume_PV";
+
+namespace chamber
+{
+const char* const wire_regex_string =
```

Comment:
why not `std::string` ? 

https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#slstr1-use-stdstring-to-own-character-sequences